AppController/djinn.rb

```diff
@@ -5878,47 +5814,27 @@ def remove_appserver_process(app_id, port)
   end
 
 
-  # Tells the AppDashboard how many requests were served for the named
-  # application at the given time, so that it can display this info to users
-  # graphically.
+  # Returns request info stored by the AppController in a JSON string
+  # containing the average request rate, timestamp, and total requests seen.
   #
   # Args:
   #   app_id: A String that indicates which application id we are storing
   #     request info for.
-  #   timestamp: An Integer that indicates the epoch time when we measured the
-  #     request rate for the given application.
-  #   request_rate: An Integer that indicates how many requests were served for
-  #     the given application in the last second since we queried it.
+  #   secret: A String that authenticates callers.
   # Returns:
-  #   true if the request info was successfully sent, and false otherwise.
-  def send_request_info_to_dashboard(app_id, timestamp, request_rate)
+  #   A JSON string containing the average request rate, timestamp, and total
+  # requests seen for the given application.
+  def get_request_info(secret, app_id)
```

Comment:
secret is usually the last parameter in the call, at least for most of the functions in djinn.